cloudflare_worker/worker.js

```diff
@@ -102,15 +110,19 @@ function rolloutGroupHeaders(request){
   return headers
 }
 
-function presortToControl(request, control_cookie_pattern, control_group){
+function presortToControl(request, control_cookie_pattern, control_group, killswitch){
+  // If the killswitch is enabled just return
+  if ( killswitch ) {
```

Comment:
Do we need this block at all?

Reply:
Nope! Was just trying to save a few calls but it really doesn't matter, can remove and reduce params going to the function